src/formatDate.js

```diff
@@ -50,7 +50,44 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const modifyDate = date.split(fromFormat[fromFormat.length - 1]);
+  const newDate = [0, 0, 0];
+  const object = {};
+  let count = 0;
+
+  let year = 0;
+
+  for (const key of fromFormat) {
```

Comment:
Name 'key' here may be a little misleading, since 'fromFormat' is an array, so it does not have keys

Reply:
Maybe using an ordinary for loop would be better, then you could get rid of the 'count' variable, and just use 'i' instead